pkg/pyinst/pyinst-build.mk

```diff
@@ -43,14 +46,12 @@ pyinst-distribution-data:
 
 pyinst-linux-helpers:
 	mkdir -p $(DIST_VERSION)apps/eip/files
-	# TODO compile static
-	cp /usr/sbin/openvpn $(DIST_VERSION)apps/eip/files/leap-openvpn
+	cp ~/leap_thirdparty_build/openvpn $(DIST_VERSION)apps/eip/files/leap-openvpn
```

Comment:
it's placed there by the scripts in the other PR. one for openvpn and the other for gnupg.
I'll modify it to use a folder inside the repo, and add it to gitignore